etc/calico-gen-bird6-mesh-conf.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+# The bird config file path is different for Red Hat and Debian/Ubuntu.
+if [ -f /etc/bird6.conf ]; then
+    BIRD_CONF=/etc/bird6.conf
+else
+    BIRD_CONF=/etc/bird/bird6.conf
+fi
+
+TEMPLATE_DIR=${TEMPLATE_DIR:-/usr/share/calico/bird}
+BIRD_CONF_TEMPLATE=${TEMPLATE_DIR}/calico-bird6.conf.template
+BIRD_CONF_PEER_TEMPLATE=${TEMPLATE_DIR}/calico-bird6-peer.conf.template
+
+# Require at least 4 arguments.
+[ $# -ge 3 ] || cat <<EOF
+
+Usage: $0 <my-ipv4-address> <my-ipv6-address> <as-number> <peer-ipv6-address> ...
+
+where
+  <my-ipv4-address> is the external IPv4 address of the local machine
+  <my-ipv6-address> is the external IPv6 address of the local machine
+  <as-number> is the BGP AS number that we should use
+  each <peer-ipv6-address> is the IPv6 address of another BGP speaker that
+      the local BIRD should peer with.
+
+EOF
+[ $# -ge 3 ] || exit -1
```

Comment:
I think I've seen negative statuses generally used.  -1 becomes 255:

    neil@nj-ubuntu:~/calico/networking-calico$ bash -c 'exit -1'
    neil@nj-ubuntu:~/calico/networking-calico$ echo $? 
    255

Anything that's non-0 is equally false.